packages/shared/components/inputs/AliasInput.svelte

```diff
@@ -26,15 +27,18 @@
             error = localize('error.aliasMinting.aliasRequired')
             return Promise.reject(error)
         }
-        const addressValidationError = validateBech32Address($networkHrp, alias, ADDRESS_TYPE_ALIAS)
-        if (addressValidationError) {
-            error = addressValidationError
-            return Promise.reject(error)
-        } else if (!isAliasInPossession()) {
-            error = localize('error.aliasMinting.aliasNotInPossession')
+
+        try {
+            validateBech32Address($networkHrp, alias, ADDRESS_TYPE_ALIAS)
+            if (isAliasInPossession()) {
+                return Promise.resolve()
+            } else {
+                error = localize('error.aliasMinting.aliasNotInPossession')
```

Comment:
you dont need to assign this value here, because it will be set when catching the error

Reply:
I addressed your comment, since Matt is away.